### ISSUE TITLE
How to strengthen the influence of the process?

### ISSUE THREAD

**Issue by roishik** [lengstrom/fast-style-transfer#146]
I implemented the code on my laptop. First of all: great code! thank you for your donation!
second: I want to strengthen the influence of the style transfer. Meaning: increase the similarity between the style image and the results.
How can I do it? which line should I change?

Thanks!

**Comment by coloboxp** [comment]
I'm having a more or less similar situation. I've even tried with the original styles you have published, but still something is lost.

Using your checkpoint works just fine, but using mine (result of training with your Style), is not so good.

![compare](https://user-images.githubusercontent.com/26008314/43364744-7b37628e-9320-11e8-846c-70cb23b580dc.png)

Please could you share with us, what fine-tunning did you do?, or which one is the knob?, or it was just more epochs?

Thank you for your time, best regards

A

**Comment by liuyanmin120** [comment]
I think is --content-weight, --style-weight, --checkpoint-iterations , these parames, I want to do this too.

**Comment by coloboxp** [comment]
Yes, would be great if the authors could share the parameters they've used, I've tried two variations at home but takes **too** long to train (10h) and I got even worse results.

**Comment by reneroth** [comment]
@lengstrom Sorry for bothering you - me and others would greatly appreciate a short write-up on the process you used to generate the provided pre-trained models.
